src/commands/malid.rs

```diff
@@ -37,28 +35,26 @@ fn format_specific_day(menu: &DailyMenu) -> Result<String, Box<dyn Error>> {
   ))
 }
 
-pub async fn run(_options: &[CommandDataOption]) -> Result<String, Box<dyn Error>> {
-  let week_menu = this_weeks_menu().await?;
-
-  format_week_menu(week_menu)
-}
-
-pub fn register(command: &mut CreateApplicationCommand) -> &mut CreateApplicationCommand {
-  command.name("lunch").description("lunch.")
-}
-
-pub async fn this_weeks_menu() -> Result<Vec<DailyMenu>, reqwest::Error> {
+async fn this_weeks_menu() -> Result<Vec<DailyMenu>, reqwest::Error> {
   let url = "https://prod-198.westeurope.logic.azure.com/workflows/cc7c4c7157b14d5ba688859712303172/triggers/manual/paths/invoke?api-version=2016-06-01&sp=%2Ftriggers%2Fmanual%2Frun&sv=1.0&sig=cRM1huMwILXk-jf6xybnCcTRpnSxjKY53jFwwUGLx14";
   let menu: Vec<DailyMenu> = reqwest::get(url).await?.json::<Vec<DailyMenu>>().await?;
 
   Ok(menu)
 }
 
+#[poise::command(slash_command)]
+pub async fn lunch(ctx: Context<'_>) -> Result<(), Error> {
+  let week_menu = this_weeks_menu().await?;
+
+  ctx.say(format_week_menu(week_menu)?).await?;
```

Comment:
```suggestion
  let msg = format_week_menu(week_menu)?;
  ctx.send(|b| {
      b.content(msg)
      .ephemeral(true)
    }).await?;
```

Reply:
Forgot to place that in, placing it back